bip-schnorr.mediawiki

```diff
@@ -84,7 +84,7 @@ The following convention is used, with constants as defined for secp256k1:
 ** ''||'' refers to byte array concatenation.
 ** The function ''x[i:j]'', where ''x'' is a byte array, returns a ''(j - i)''-byte array with a copy of the ''i''-th byte (inclusive) to the ''j''-th byte (exclusive) of ''x''.
 ** The function ''bytes(x)'', where ''x'' is an integer, returns the 32-byte encoding of ''x'', most significant byte first.
-** The function ''bytes(P)'', where ''P'' is a point, returns ''bytes(0x02 + (y(P) & 1)) || bytes(x(P))''<ref>This matches the ''compressed'' encoding for elliptic curve points used in Bitcoin already, following section 2.3.3 of the [http://www.secg.org/sec1-v2.pdf SEC 1] standard.</ref>.
+** The function ''bytes(P)'', where ''P'' is a point, returns ''byte(0x02 + (y(P) & 1)) || bytes(x(P))''<ref>This matches the ''compressed'' encoding for elliptic curve points used in Bitcoin already, following section 2.3.3 of the [http://www.secg.org/sec1-v2.pdf SEC 1] standard.</ref>.
```

Comment:
Hm, yes! It may look like a typo though because `byte` is not defined but `bytes` is. Maybe just:
```suggestion
** The function ''bytes(P)'', where ''P'' is a point, returns ''(0x02 + (y(P) & 1)) || bytes(x(P))''<ref>This matches the ''compressed'' encoding for elliptic curve points used in Bitcoin already, following section 2.3.3 of the [http://www.secg.org/sec1-v2.pdf SEC 1] standard.</ref>.
```
or even more readable:
```suggestion
** The function ''bytes(P)'', where ''P'' is a point, returns ''0x02 || bytes(x(P))'' if ''y(P) & 1 = 0'' and ''0x03 || bytes(x(P))'' otherwise.<ref>This matches the ''compressed'' encoding for elliptic curve points used in Bitcoin already, following section 2.3.3 of the [http://www.secg.org/sec1-v2.pdf SEC 1] standard.</ref>.
```

Reply:
I like the first suggestion